package.json

```diff
@@ -61,7 +61,8 @@
     "tslib": "1.10.0",
     "vscode": "^1.1.36",
     "vue": "3.0.0-rc.10",
-    "ws": "7.0.0"
+    "ws": "7.0.0",
+    "cross-fetch": "3.0.6"
```

Comment:
@mi-kas Hello my dear friend!
Do we still need this line? I've seen you managed to get rid of `cross-fetch` dependency by bumping `graphql-request` version.

Reply:
@SergeiKalachev Good catch my friend 👍  Removed now this dependency